### PR TITLE
Add support for VPN CTA for Contentful Hero components

### LINKED ISSUE #10447: Get Mozilla VPN link on Contentful home page goes to getpocket.com
## Description

The "Get Mozilla VPN" link on Contentful home page goes to `getpocket.com` instead of `/products/vpn/`.

### Steps to reproduce

1. https://www-dev.allizom.org/en-US/
2. Scroll down the the "Protection for your whole device, on every device" section.
3. Click "Get Mozilla VPN".

Screenshot: 

image

### Expected result

The page should navigate to https://www-demo1.allizom.org/en-US/products/vpn/

### Actual result

The link goes to https://getpocket.com/?utm_source=www.mozilla.org-home&utm_medium=referral&utm_campaign=home

### Environment

N/A

### PR DESCRIPTION
**Pull request by pmac**
Fix #10447

### CODE REVIEW

**Comment by pmac** [comment]
This should, after we add the "Get Mozilla VPN" action as a choice in the admin, fix this issue. We should probably think harder about how we'd like to enable this kind of thing more robustly in future though.

File: bedrock/contentful/templates/includes/contentful/cta.html [hunk 1]
```diff
@@ -22,7 +22,13 @@
   ) }}
 
 {% elif action == 'Get Pocket' %}
+
     {% set referral = '?utm_source=' + utm_source + '&utm_medium=referral&utm_campaign=' + utm_campaign %}
     <a href="https://getpocket.com/{{ referral|safe }}" class="mzp-c-button {{ button_class }}" rel="external noopener" data-cta-type="link" data-cta-text="{{ cta_text }}">{{ label }}</a>
+
+{% elif action == 'Get Mozilla VPN' %}
+
+  <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button {{ button_class }}">{{ label }}</a>
```
Comment (alexgibson) [inline bedrock/contentful/templates/includes/contentful/cta.html:31]: Suggest adding some data attributes so we can see the numbner of clicks this gets in GA?
```suggestion
  <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button {{ button_class }}" data-cta-text="{{ cta_text }}" data-cta-type="button" data-cta-position="{{ location }}">{{ label }}</a>
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
Looks good - one suggested change to make sure these buttons get events in GA.

File: bedrock/contentful/templates/includes/contentful/cta.html (hunk 1 above)
Comment (pmac) [reply bedrock/contentful/templates/includes/contentful/cta.html:31]: Thanks! I literally just copied the code from the current VPN CTA on the home page.

Comment (pmac) [reply bedrock/contentful/templates/includes/contentful/cta.html]: I do see above this one that the `data-cta-type` for Pocket is `link` and not `button`. They're both buttons though. They should be the same type right?

Comment (alexgibson) [reply bedrock/contentful/templates/includes/contentful/cta.html]: Good question. We seem to use these pretty interchangably and there's no real definition for `link` as a value in our docs: https://bedrock.readthedocs.io/en/latest/analytics.html#gtm-listeners-data-attributes

To err on the side of consistency I'd use `link` as well here. Although throughout the site as a whole I see a good mixture of both.

Comment (pmac) [reply bedrock/contentful/templates/includes/contentful/cta.html]: Cool. Done.

**Review by alexgibson — APPROVED** [review]

**Pull request merged**